examples/dub.json

```diff
@@ -9,7 +9,7 @@
             "sourceFiles": ["tutorial/addressbook.d"],
             "sourcePaths": ["../src"],
             "mainSourceFile": "add_person.d",
-            "preBuildCommands": ["protoc $$PROTO_PATH --plugin=../build/protoc-gen-d --d_out=. addressbook.proto"]
+            "preBuildCommands": ["protoc $$PROTO_PATH --plugin=../build/protoc-gen-d --d_opt=make-structs --d_out=. addressbook.proto"]
```

Comment:
`make-struct` is not self-explaining, I would prefer something like `message-as-struct` or even `define-message-as-struct`

Reply:
agree, renamed